### ISSUE TITLE
Bridge in Ro'maeve dissapearing [and too high]

### ISSUE THREAD

**Issue by kaincenteno** [project-topaz/topaz#480]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
Two weird things are happening with the bridges:
* the highest bridge (the one right next to the fountain), dissapears entirely if you walk away from it by at least 50 meters.
* they raise a bit too much when night time on a full moon. Unsure if its retail accurate or if pos needs to decrease a bit.

**Comment by TeoTwawki** [comment]
> the highest bridge (the one right next to the fountain), dissapears entirely if you walk away from it by at least 50 meters.

Entity packets not currently sent when farther than that. For most objects that should be retail accurate but then when you come in range they should "repop" on your screen. may need a way to make exceptions to that. Not sure what retail does in this case for these specific objects.

> hey raise a bit too much when night time on a full moon. Unsure if its retail accurate or if pos needs to decrease a bit.

Are you able to step onto them?

**Comment by kaincenteno** [comment]
@TeoTwawki 
yeah once u get back to it the whole bridge is gone lol. it only reappears once the moon gate doors close

Yes you are able to step on them but it's raise too high it makes the taru feet dissapear lol

**Comment by TeoTwawki** [comment]
Its eating taru feet? clearly a secret mimic, and a racial hate crime.  We can't let that stand. To war! [We'll show those packets!](https://github.com/project-topaz/topaz/blob/4d09173b57dd5867dd3997436fa3ee66eeea0fbb/src/map/zone_entities.cpp#L787)

**Comment by TeoTwawki** [comment]
https://github.com/project-topaz/topaz/blob/4d09173b57dd5867dd3997436fa3ee66eeea0fbb/src/map/zone_entities.cpp#L787

**Comment by kaincenteno** [comment]
@TeoTwawki  hopefully someone takes a look at this =) funny thing is that it only happens to the top bridge but not the bottom.

This should help also with the gates in the desert (those high towers) for opening that door that u need to hit on 4 switches in the map. cause if u are far away the look as if they are off, but if u get close they look as if the switch is on
